### ISSUE TITLE
Version are sorted by string rather than number ? Version 9.2 is larger than 11.6 !

### ISSUE THREAD

**Issue by lpatiny** [google-github-actions/release-please-action#499]
### TL;DR

The following project https://github.com/mljs/spectra-processing is current at version 11.6.0.

We are using a 'release-please-action' in this github action:

https://github.com/mljs/spectra-processing/blob/master/.github/workflows/release.yml

that uses the following template:

https://github.com/zakodium/workflows/blob/main/.github/workflows/release.yml

In the latest PR rather than adding the changes since 11.6.0, release-please added all the changes since 9.2.0 and has proposed an lower version number

https://github.com/mljs/spectra-processing/pull/153/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

### Expected behavior

We expect that only changes from 11.6.0 appears in the change log and the new version is 11.7.0

### Observed behavior

All the changes since 9.2.0 are in the change log and the proposed version is 10.0.0

### Action YAML

```yaml
name: Release

on:
  push:
    branches:
      - master

jobs:
  release:
    uses: zakodium/workflows/.github/workflows/release.yml@release-v1
    with:
      npm: true
    secrets:
      github-token: ${{ secrets.BOT_TOKEN }}
      npm-token: ${{ secrets.NPM_BOT_TOKEN }}

---------------------------------------------------

name: Release a Node.js package

on:
  workflow_call:
    inputs:
      npm:
        description: >
          Whether the package will be published to npm. You must pass the
          npm-token secret in that case.
        type: boolean
        default: false
      node-version:
        description: Version of Node.js used to run the npm publish steps.
        type: string
        default: 16.x
      npm-setup-command:
        description: Command used to setup the package before publishing to npm.
        type: string
        default: npm install
    secrets:
      github-token:
        required: true
      npm-token:
        required: false

jobs:
  release-please:
    runs-on: ubuntu-latest
    outputs:
      release_created: ${{ steps.release.outputs.release_created }}
    steps:
      - name: Get package name
        run: echo "PACKAGE_NAME=$(jq .name package.json | tr -d '"')" >> $GITHUB_ENV
      - uses: google-github-actions/release-please-action@v3
        id: release
        with:
          token: ${{ secrets.github-token }}
          release-type: node
          package-name: ${{ env.PACKAGE_NAME }}
          bump-minor-pre-major: true
  npm-publish:
    needs: release-please
    if: ${{ inputs.npm && needs.release-please.outputs.release_created }}
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version: ${{ inputs.node-version }}
          registry-url: 'https://registry.npmjs.org'
      - run: ${{ inputs.npm-setup-command }}
      - run: npm publish --access public
        env:
          NODE_AUTH_TOKEN: ${{ secrets.npm-token }}
```

**Comment by lpatiny** [comment]
This was fixed in https://github.com/googleapis/release-please/pull/1451, thanks !
